nemo/collections/tts/models/glow_tts.py

```diff
@@ -205,14 +211,18 @@ def validation_epoch_end(self, outputs):
 
     def _setup_dataloader_from_config(self, cfg: DictConfig):
 
+        if 'manifest_filepath' in cfg and cfg['manifest_filepath'] is None:
+            logging.warning(f"Could not load dataset as `manifest_filepath` was None. Provided config : {cfg}")
+            return None
+
         if 'augmentor' in cfg:
             augmentor = process_augmentations(cfg['augmentor'])
         else:
             augmentor = None
 
-        dataset = AudioToPhonemesDataset(
+        dataset = _AudioTextDataset(
```

Comment:
I dont think you are meant to be using `_AudioTextDataset`

Reply:
Let's revert back to `AudioToPhonemesDataset`, I'm sure other TTS and ASR models will need it in the future.

Reply:
I changed it to that because the only difference was the parser used, which actually handles the phoneme processing